.github/prlint.json

```diff
@@ -0,0 +1,32 @@
+{
+    "title": [
+        {
+            "pattern": "^(docs|feat|fix|dev|test):\\s",
+            "message": "Your title needs to be prefixed with a topic"
+        }
+    ],
+    "body": [
+        {
+            "pattern": ".{1,}",
+            "message": "You need to have some content in the description."
+        }
+    ],
+    "head.ref": [
+        {
+            "pattern": "^(docs|feat|fix|dev|test)/",
+            "message": "Your branch name is invalid. Please prefix it with docs|feat|fix|dev|test."
+        }
+    ],
+    "assignee.login": [
+        {
+            "pattern": ".+",
+            "message": "You need to assign someone"
```

Comment:
Not sure about this one. 
I guess for PRs they would assign themselves?
I guess is not necessary! Thoughts?

Reply:
I think it would be great if they assign themselves so that we know who those commits belong to.

Reply:
But we know it already, no?
We have that info in the commits themselves, we won't do anything with this information.